packages/frontend/src/components/organisms/auth/RegisterTab.tsx

```
@@ -36,69 +38,48 @@ export function RegisterTab({ onSubmit, isLoading, error }: RegisterTabProps) {
-            <Tooltip content="Der Benutzername muss mit einem Buchstaben beginnen und darf nur Buchstaben, Zahlen und Unterstriche enthalten.">
-              <span style={{ marginLeft: '8px', cursor: 'help' }}>
+        <FormField
+          label={
+            <span className="flex items-center gap-2">
+              Neuer Benutzername
+              <span
+                title="Der Benutzername muss mit einem Buchstaben beginnen und darf nur Buchstaben, Zahlen und Unterstriche enthalten."
+                aria-label="Info: Regeln für den Benutzernamen"
+                className="cursor-help text-gray-500 transition-colors hover:text-gray-700"
+              >
                 <PiInfo />
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)